OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -70,6 +72,19 @@ void IRenderAboveWorld.RenderAboveWorld(Actor self, WorldRenderer wr)
 				TargetLineRenderable.DrawTargetMarker(wr, hc, hb);
 			}
 
+			var tc = Color.Lime;
+			var enabledPositions = targetablePositions.Where(Exts.IsTraitEnabled);
+			var positions = enabledPositions.SelectMany(tp => tp.TargetablePositions(self));
+			foreach (var p in positions)
+			{
+				var updown = new WVec(0, 128, 0);
+				var leftright = new WVec(128, 0, 0);
```

Comment:
Caching would be pointless for value types. In fact, that would increase the chances of a cache miss depending on where it is memory and it would just be copied anyways.
See https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-structs.
However, a case could be made that these are a kind of constant and should be `static readonly`.